### ISSUE TITLE
Quick Access Widget

### ISSUE THREAD

**Issue by ageev** [andOTP/andOTP#616]
I need to put OTP code from my phone to PC like 10 times per day for one single website. 
Quick Access widget which will reveal the  code for particular app in one tap will really help me. 

Currently I need to open andOTP app, find needed app and click on it to do that (so 2 clicks + visual search). 
Widget will make it in 1 click without search.

I would like to see andOTP Quick Access widget. When adding this widget to the desktop, it should ask me what app do I want to see OTP codes from. After clicking the widget I should see the popup with OTP code or OTP code should be displayed on the widget itself for few seconds. Some users may even create a folder with quick access widgets for most frequently used app. In that scenario this folder can act as "favorites".

PS thank you for the wonderful app 👍

**Comment by beerisgood** [comment]
You can search in andOTP and last used ones are at stop so that is still to hard?

**Comment by ageev** [comment]
no, not hard at all, but widget will make it lighting fast (1 click, or even 0 click with "always show" option). 
That's the whole idea of widgets, right? Give you the feature/info instantly without need to open the app.

**Comment by beerisgood** [comment]
I get what you mean but this would also reduce the security and 2FA is enhanced security

**Comment by ageev** [comment]
Don't mix everything into one bucket please. 2fa means that there are 2 authentication secrets phisically splitted one from another. Widget doesn't brake this scheme in any possible way. 

Widget removes local authentication from OTP codes, but this is user's choice and currently already implemented in the app ("authentication" parameter could be set to "none" in app settings).
